CONTRIBUTING.md

```diff
@@ -42,8 +44,90 @@ mvn -f pom.client.xml -Dgpg.skip -DskipTests clean install
 >**Note**: Refer to [wiki](https://github.com/Azure/azure-sdk-for-java/wiki/Building) for learning about how to build using Java 11
 
 ### Compiling one project only
+
 ```
 mvn -f sdk/{projectForlderDir}/pom.xml -Dgpg.skip clean install
 
 //example: mvn -f sdk/keyvault/azure-keyvault-keys/pom.xml clean install
 ```
+
+## Versions and versioning
+
+Tooling has been introduced to centralize versioning and help ease the pain of updating artifact versions in POM and README files. Under the eng\versioning directory there version text files, one for client (version_client.txt) and one for data (version_data.txt). The format of the version files is as follows:
+groupId:artifactId;dependency-version;current-version
+The dependency-version should be set to the most recent released version and the current-version is set to the next version to be released.
```

Comment:
It doesn't matter too much to me, but "dependency-version" might be better named "current-version", and "current-version" might be better named "next-version".

Reply:
It's funny as I'd had several talks around this before agreeing on the convention being used and settled on this. Right now this is going to stay the way it is.